docs/docs-site/package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
 
-    "clean-static": "rm -rf static/js/gethue/*",
-    "copy-static": "cp -r node_modules/gethue/components static/js/gethue/ && cp -r node_modules/gethue/parsers static/js/gethue/",
-    "update-static": "npm run clean-static && npm run copy-static"
+    "clean-static": "rm -rf static/js/gethue",
+    "copy-components": "mkdir -p static/js/gethue/components && cp -r node_modules/gethue/lib/components/*.js static/js/gethue/components/",
```

Comment:
Where are we documenting these changes? I.e. how does a new developer know when these commands needs to be run and why?  There is a very short section on `docs/docs-site/README.md` and I think that would be a good place to expand on this.

Reply:
Readme is still valid, no change in commands for whomever is trying to update - clean-static & copy-components is run by update-static.

Reply:
Good point